src/transformers/convert_slow_tokenizer.py

```diff
@@ -635,9 +635,14 @@ def vocab(self, proto):
         vocab = [
             (self.original_tokenizer.pad_token, 0.0),
             (self.original_tokenizer.eos_token, 0.0),
-            (self.original_tokenizer.mask_token_sent, 0.0),
-            (self.original_tokenizer.mask_token, 0.0),
         ]
+
+        if self.original_tokenizer.mask_token_sent is not None:
+            vocab += [
+                (self.original_tokenizer.mask_token_sent, 0.0),
+                (self.original_tokenizer.mask_token, 0.0),
```

Comment:
Should we check for `self.original_tokenizer.mask_token` separately?

Reply:
Good point! I'll make this change